AnkiDroid/src/main/java/com/ichi2/libanki/Sched.java

```diff
@@ -628,17 +628,20 @@ private Card _getNewCard() {
     }
 
 
-    private void _updateNewCardRatio() {
+    private void _updateNewCardRatio(int criticalReviewCount) {
```

Comment:
@wclausen numbers here are just guesses at what makes sense, we can adjust as we gain experience. Overall the idea is, the system should automatically show you less new cads if your critical backlog is massive

Reply:
This is sick. What's here seems like a good starting point and pretty simple.

Reply:
Yep, verified it already works pretty well. If anything, it's actually still showing new cards a bit more frequently than we'd like. I've updated it to be 2x less frequent now.